src/System.Private.Xml/tests/System.Private.Xml.Tests.builds

```diff
@@ -2,102 +2,112 @@
+    <Project Include="Writers\XmlWriterApi\System.Xml.RW.XmlWriterApi.Tests.csproj" />
     <Project Include="Writers\XmlWriterApi\System.Xml.RW.XmlWriterApi.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="Writers\XmlWriterApi\System.Xml.RW.XmlWriterApi.Tests.csproj" />
+    <Project Include="XmlConvert\System.Xml.RW.XmlConvert.Tests.csproj" />
     <Project Include="XmlConvert\System.Xml.RW.XmlConvert.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlConvert\System.Xml.RW.XmlConvert.Tests.csproj" />
+    <Project Include="XmlDocument\System.Xml.XmlDocument.Tests.csproj" />
     <Project Include="XmlDocument\System.Xml.XmlDocument.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlDocument\System.Xml.XmlDocument.Tests.csproj" />
+    <Project Include="XmlReaderLib\System.Xml.RW.XmlReaderLib.csproj" />
     <Project Include="XmlReaderLib\System.Xml.RW.XmlReaderLib.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlReaderLib\System.Xml.RW.XmlReaderLib.csproj" />
+    <Project Include="XmlReader\ReadContentAs\System.Xml.RW.XmlReader.ReadContentAs.Tests.csproj" />
     <Project Include="XmlReader\ReadContentAs\System.Xml.RW.XmlReader.ReadContentAs.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlReader\ReadContentAs\System.Xml.RW.XmlReader.ReadContentAs.Tests.csproj" />
+    <Project Include="XmlReader\Tests\System.Xml.RW.XmlReader.Tests.csproj" />
     <Project Include="XmlReader\Tests\System.Xml.RW.XmlReader.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlReader\Tests\System.Xml.RW.XmlReader.Tests.csproj" />
+    <Project Include="XmlReader\XmlResolver\System.Xml.RW.XmlSystemPathResolver.Tests.csproj" />
     <Project Include="XmlReader\XmlResolver\System.Xml.RW.XmlSystemPathResolver.Tests.csproj">
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
-    <Project Include="XmlReader\XmlResolver\System.Xml.RW.XmlSystemPathResolver.Tests.csproj" />
-    <Project Include="XmlSchema\XmlSchemaSet\System.Xml.XmlSchemaSet.Tests.csproj" />
-    <Project Include="XmlSchema\XmlSchemaValidatorApi\System.Xml.XmlSchema.XmlSchemaValidatorApi.Tests.csproj" />
+    <Project Include="XmlSchema\XmlSchemaSet\System.Xml.XmlSchemaSet.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
`XmlSchema` and `Xsl` tests need changes before they can run successfully on non-Windows platforms (as they are using file paths and URIs). I created issue https://github.com/dotnet/corefx/issues/12789 for that.